tests/test_calibration.py

```diff
@@ -60,25 +60,25 @@ def test_CCDunit():
     with open('testdata/CCDunit_UV1_example.pkl', 'wb') as f:
         pickle.dump(CCDunit_IR1, f)
 
-def test_forward_backward(): 
-    """
-    This tests the forward and backward calibration. 
-    The backward calibraton should completely reverse everything the forward 
-    calibration has done thus giving back the original image.
-    This test needs a CCDitem and a CCDunit, which are created and saved in 
-    test_reafunctions and test_CCDunit.
+# def test_forward_backward(): 
```

Comment:
Why has this been commented out? Was it committed by mistake?

If it is (currently) expected to fail, an alternative is to mark it like this:
```suggestion
@pytest.mark.xfail(reason="reason for failure, maybe reference an issue number)
def test_forward_backward(): 
```